go-controller/pkg/node/gateway_iptables.go

```diff
@@ -328,16 +348,24 @@ func initLocalGatewayNATRules(ifname string, cidr *net.IPNet) error {
 	return addIptRules(getLocalGatewayNATRules(ifname, cidr))
 }
 
+func addChaintoTable(ipt util.IPTablesHelper, tableName, chain string) {
+	if err := ipt.NewChain(tableName, chain); err != nil {
+		klog.V(5).Infof("Chain: \"%s\" in table: \"%s\" already exists, skipping creation", chain, tableName)
```

Comment:
yeah I think that is the only error :D
```
	// NewChain creates a new chain in the specified table.
	// If the chain already exists, it will result in an error.
	NewChain(string, string) error
```
but I can add %v, err for more information.